pebblo/app/models/models.py

```diff
@@ -147,3 +148,22 @@ class AppModel(BaseModel):
     findings: list
     documentsWithFindings: list
     dataSource: list
+
+
+class LoaderDocs(BaseModel):
```

Comment:
Why do we need a new model when we already have a similar model 'AiDocs'?

Reply:
The internal facing model has fields with camel case. The fields in response would be in the snake case. That's the only difference.